data-es6.js

```diff
@@ -3586,6 +3586,54 @@ exports.tests = [
         iojs:        true,
       },
     },
+    'constructor accepts null': {
```

Comment:
Please add `es6shim: true` to this test, since the `es6-shim`'s `Map` constructor does accept null.